<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="button_sto1">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>341</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sto 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_sto2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>240</y>
     <width>341</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sto 2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
